pkg/scheduler/api/devices/nvidia/vgpu4pd/device_info.go

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
please change the directory name from `vgpu4pd` to `vgpu` to make it more common.

Reply:
fixed